src/gui/transferlistfilterswidget.cpp

```diff
@@ -574,16 +574,25 @@ TransferListFiltersWidget::TransferListFiltersWidget(QWidget *parent, TransferLi
     scroll->setWidget(frame);
     vLayout->addWidget(scroll);
     setLayout(vLayout);
-
-    QCheckBox *statusLabel = new QCheckBox(tr("Status"), this);
+		
+		auto IconCheckBox = [&](auto &&name) {
```

Comment:
And please remove this lambda, it is overkill for what you're doing.

Reply:
what do you suggest, calling setStylesheet for each Checkbox?

Reply:
>calling setStylesheet for each Checkbox?

yes.